packages/snaps-utils/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Changed
+- **BREAKING:** Update ESM build to be fully compliant with the ESM standard ([#2210](https://github.com/MetaMask/snaps/pull/2210))
+- **BREAKING:** Move Node.js exports to separate export ([#2210](https://github.com/MetaMask/snaps/pull/2210))
+  - The default export is now browser-compatible.
+  - Node.js APIs can be imported from `<package>/node`.
+- **BREAKING:** Move `file` struct to CLI ([#2207](https://github.com/MetaMask/snaps/pull/2207))
+  - The previously exported `file` struct can now be found in `@metamask/snaps-cli`.
```

Comment:
Should this be under "Removed"?

Reply:
Could be, we've also put these under "Changed" in the past.

But I will move it